Sources/Rendering/Core/Renderer/index.js

```diff
@@ -597,7 +597,7 @@ export function extend(publicAPI, model, initialValues = {}) {
     'pass',
   ]);
   macro.getArray(publicAPI, model, ['actors', 'volumes', 'lights']);
-  macro.setGetArray(publicAPI, model, ['background'], 3);
+  macro.setGetArray(publicAPI, model, ['background'], [3, 4]);
```

Comment:
I could be wrong, but looking at `macro.js`, I don't think this is going to work.  It seems like you were thinking of supporting background setting which could take 3 or 4 element array?  Another way would be just creating a background opacity property that would default to 1.0, but could be set to something else.